ui/00_message/tutorial_guide/tutorial_guide.xml

```diff
@@ -920,6 +921,7 @@
   <message>
     <key>TUTORIAL_365</key>
     <original>ボーナスダンジョン</original>
+    <translation>Bonus Dungeon</translation>
```

Comment:
Please remove this or make it plural (I have a local change for this that will conflict if left as-is).